### ISSUE TITLE
Upgrade to Spring Boot 2

### ISSUE THREAD

**Issue by davsclaus** [redhat-developer-demos/istio-tutorial#141]
Would it not be good to update to SB2 ?

Apache Camel 2.22.x supports SB2 so you should be able to easily upgrade to SB2 and Camel at the same time. At this time of writing Camel 2.22.1 is the latest release.

**Comment by rafabene** [comment]
Indeed. However we are focusing more on the Istio features than the implementation itself. When we update the maven dependencies, we need also to regenerate the katacoda image that runs at https://learn.openshift.com/servicemesh

**Comment by rafabene** [comment]
Now that Quarkus is live, does this issue make sense?

**Comment by davsclaus** [comment]
Well yes and no. SB is very well known and in use. By having SB and other other stacks we can show that we can work with all of the popular ones, and people can then start migrate their apps with SB to start using them with Istio and then learn that there is a better java runtime such as quarkus and then start using it etc.

**Comment by davsclaus** [comment]
But if you remove SB then you should remove it also in the title of the root readme :)

**Comment by rafabene** [comment]
Done :)

**Comment by rafabene** [comment]
Can you contribute to PR?
